specs/v2-sketch.yaml

```diff
@@ -303,6 +348,22 @@ components:
         last_job_finish_time:
           type: string
           format: date-time
+    TriggerPipelineRequest:
+      type: object
+      additionalProperties: false
+      properties:
+        branch:
+          description: The branch to run the new pipeline on. The HEAD commit on this branch will be used for the triggered pipeline, unless the `revision` parameter is also passed.
+          type: string
+          example: "feature/design-new-api"
+        revision:
+          description: The revision of code that the new pipeline is to run.
+          type: string
+          example: f454a02b5d10fcccfd7d9dd7608a76d6493a98b4
+        tag:
+          description: A tag identifying the revision of the code that the new pipeline is to run. Note that `tag` cannot be specified with `revision` or `branch`.
+          type: string
+          example: "v2.0"
```

Comment:
It might be worthwhile to pick a tag value like `"v3.1.4"` which is obviously nothing to do with our own versions.